### PR TITLE
Get and set wallpapers on GNOME in Linux

### PR DESCRIPTION
**Pull request by naveenkumarsangi**
* Also changed error messages in GetWallpaper() and SetWallpaper()
* It works

### CODE REVIEW

File: utils/linux.go [hunk 1]
```diff
@@ -1 +1,44 @@
 package utils
+
+import (
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// GetMacOSWallpaper returns the path to the current wallpaper.
```
Comment (naveenkumarsangi) [inline utils/linux.go]: Please run lint checks before submission. `go list ./... | grep -v /vendor/ | xargs -L1 golint`

File: utils/linux.go [hunk 2]
```diff
@@ -1 +1,44 @@
...
+	).Output()
+
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(string(stdout)), nil
+}
+
+// SetMacOSWallpaper sets the wallpaper to the picture from provided path.
```
Comment (naveenkumarsangi) [inline utils/linux.go]: Same goes here.

File: utils/wallpaper.go [hunk 3]
```diff
@@ -22,7 +24,9 @@ func SetWallpaper(path string) error {
 
 	if runtime.GOOS == "darwin" {
```
Comment (naveenkumarsangi) [inline utils/wallpaper.go]: maybe it's better using a switch case with default.

**Comment by naveenkumarsangi** [comment]
Can we rather have an [editorconfig?](http://editorconfig.org)

**Comment by Compro-Prasad** [comment]
I don't think we need it. Because `go fmt` is enough to do the stuff. I just added something so that Emacs users(if any and including me) could get the tab width of **4** spaces when opening a file in this project.

**Comment by naveenkumarsangi** [comment]
:P what about people using other text editors? EditorConfig unifies all of the needs.

**Comment by naveenkumarsangi** [comment]
also, I think we should rather take care of conditional builds rather than having two functions. Can you cherry pick and squash the commits together?

**Comment by Compro-Prasad** [comment]
I don't know how editorconfig will perform with Emacs because different languages, modes and buffers use different variables to set styling properties. Its better if we can try it out.

File: utils/wallpaper.go [hunk 4]
```diff
@@ -1,28 +1,19 @@
+// +build !darwin,!linux
```
Comment (naveenkumarsangi) [inline utils/wallpaper.go:1]: Makes no sense to have this file.

File: .dir-locals.el [hunk 5]
```diff
@@ -0,0 +1,12 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
```
Comment (naveenkumarsangi) [inline .dir-locals.el]: Like as I always said 😛

We need an EditorConfig, that too in a separate commit, please.

File: utils/wallpaper.go (hunk 4 above)
Comment (Compro-Prasad) [reply utils/wallpaper.go:1]: Other users might try to build this project. So, I am making sure it builds successfully but doesn't provide the feature.

**Comment by naveenkumarsangi** [comment]
Please rebase your branch with master so that this PR can be merged. Also check for failing CI builds.

**Comment by Compro-Prasad** [comment]
Is the PR okay now?

**Pull request merged**